pipelines/deployer/deployer.yaml

```
@@ -337,6 +338,44 @@ check_cloudwatch: &check_cloudwatch
       echo "FAIL: No logs have been detected reaching cloudwatch since $LOGS_SINCE"
       exit 1
 
+check-health-monitoring: &check-health-monitoring
+  platform: linux
+  image_resource: *task_image_resource
+  params:
+    ACCOUNT_ROLE_ARN: ((account-role-arn))
+    AWS_REGION: eu-west-2
+    AWS_DEFAULT_REGION: eu-west-2
+    CLUSTER_NAME: ((cluster-name))
+  run:
+    path: /bin/bash
+    args:
+      - -eu
```

```suggestion
      - -euo
```

Task errors with:

> /bin/bash: line 0: /bin/bash: -c: invalid option name

if this isn't split out properly 🤔